docs/events.md

```diff
@@ -30,25 +30,21 @@ Timing events log the duration that a specific action took plus some metadata th
 
 #### Window load time ([more info](https://atom.io/docs/api/v1.35.1/AtomEnvironment#instance-getWindowLoadTime))
 
-* **eventType**: `core`
+* **eventType**: `load`
```

Comment:
I haven't changed the behaviour here (the event type was already `load` before ,as seen in [this comment](https://github.com/github/central/pull/430#issuecomment-474479944)), it just wrongly documented (the redirection of using the `sendTiming` method with different order of arguments made it really confusing...